source/appendices/e-audit-requirements-and-tests/04-dr-tests.rst

```diff
@@ -0,0 +1,66 @@
+
+Audit test requirements vary depending on the set up of the platform and database(s), as indicated by the Options within the tests below.
+
+Database Report tests apply to Host_Types A&I_Database, Aggregated_Full_Content, Discovery_Service, eBook_Collection, Full_Content_Database, and Multimedia_Collection.
+
+E.4.1 Searches_Regular and Searches_Automated
+"""""""""""""""""""""""""""""""""""""""""""""
+
+**Option 1**: Platform has multiple databases and the user can search the whole platform, multiple selected databases, or a single database.
+
+The auditor MUST run 100 searches on the platform
+
+* 50 searches against only 1 selected database, resulting in 50 Searches_Regular against that database.
+* 25 searches against 2 selected databases, resulting in 25 Searches_Regular against both databases.
+* 25 searches against all databases on the platform, resulting in 25 Searches_Regular against every database.
+
+**Option 2**: Platform has multiple databases and the user can search the whole platform or a single database.
+
+The auditor MUST run 100 searches on the platform
+
+* 50 searches against only 1 selected database, resulting in 50 Searches_Regular against that database.
+* 50 searches against all databases on the platform, resulting in 50 Searches_Regular against every database.
+
+**Option 3**: Platform has multiple databases but the user can only search the whole platform.
+
+The auditor MUST run 100 searches, resulting in 100 Searches_Automated (1 per search).
+
+**Option 4**: Platform has multiple databases but the user can only search a single database at a time.
+
+The auditor MUST run 100 searches across at least 10% of the available databases, restricted to one database at a time, resulting in 100 Searches_Regular (1 per search).
+
+**Option 5**: Platform has only one database.
+
+The auditor MUST run 50 searches, resulting in 50 Searches_Regular (1 per search).
+
+
+E.4.2 Total_Item_Requests and Unique_Item_Requests
+""""""""""""""""""""""""""""""""""""""""""""""""""
+
+Where possible, 50% of items requested SHOULD be via browsing the platform and 50% via searching the platform. If either browsing or searching is not possible, all items can be requested via the only available option.
+
+The auditor MUST make a total of 80 requests against different items, resulting in 80 each of Total_Item_Investigations, Total_Item_Requests, Unique_Item_Investigations and Unique_Item_Requests.
+
+Where a platform has fewer than 80 items, the auditor MUST make at least one request per item and testing should result in 80 each of Total_Item_Investigations and Total_Item_Requests, and the item-count of Unique_Item_Investigations and Unique_Item_Requests.
```

Comment:
It can also happen for other tests that there a fewer items that tests required (e.g. in E.3.2). Maybe add a generic note how this case should be handled and remove this paragraph...

Reply:
See addition to section 1, audit-requirements